internal/cmd/cli.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+)
+
+// CLI exposes common dependencies to commands.
+type CLI struct {
```

Comment:
I expect we will very likely add to this type. Probably common CLI options  and maybe an `api.Client`.

We should also do something with logging. Right now patching `stderr` doesn't do much because the logger would have already grabbed a reference to `os.Stderr` before this is called.